bucket/etcher.json

```
@@ -1,25 +1,42 @@
 {
-    "homepage": "https://etcher.io/",
+    "homepage": "https://www.balena.io/etcher/",
     "license": "Apache-2.0",
     "version": "1.5.50",
+    "description": "Flash OS images to SD cards & USB drives, safely and easily.",
     "url": "https://github.com/balena-io/etcher/releases/download/v1.5.50/balenaEtcher-Setup-1.5.50.exe#/dl.7z",
-    "hash": "8fedaf6c0556eeb2b4f9df6f1813d02934b40fca5f1a4feb87fa3d479b6b5d3a",
+    "hash": "sha512:c404b7442f62c9e18408cffc6e22b3de995f2bc2f9c3a53af7ca56d850cbdb9cfa5a99fe7a9723884dd9f9e8f60bf0ec52c91c0459cbab44afb536ec4a0759f6",
+    "architecture": {
+        "64bit": {
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
+                ]
+            }
+        },
+        "32bit": {
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\"",
```

https://github.com/lukesampson/scoop/blob/69a0843e9a1f1bf5d3b179beea86fbda44d0ec4d/lib/decompress.ps1#L44-L46

```suggestion
                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\"",
```

---

I removed the `\"$dir\"`, but the needed files weren't then copied to $dir